packages/trader/src/App/Components/Elements/Errors/error-component.tsx

```diff
@@ -1,24 +1,32 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Dialog, PageErrorContainer } from '@deriv/components';
 import { routes } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 
-const ErrorComponent = ({
+type TErrorComponent = {
+    header: string;
+    message: React.ReactNode;
+    is_dialog: boolean;
+    redirect_label: string;
+    redirectOnClick: () => void;
+    should_show_refresh: boolean;
+};
+
+const ErrorComponent: React.FC<Partial<TErrorComponent>> = ({
     header,
     message,
     is_dialog,
     redirect_label,
     redirectOnClick,
     should_show_refresh = true,
 }) => {
-    const refresh_message = should_show_refresh ? localize('Please refresh this page to continue.') : '';
+    const refresh_message: string = should_show_refresh ? localize('Please refresh this page to continue.') : '';
 
     if (is_dialog) {
         return (
             <Dialog
                 title={header || localize('There was an error')}
-                is_visible
+                is_visible={true}
```

Comment:
Why do we need this?

Reply:
This is to explicitly define the value inside, it's the same with or without it, I just added for better readability